### PR TITLE
Alien Plant, Leaker

### PR DESCRIPTION
**Pull request by Sovexe**
## About the PR 
* Adds a new dripper mutation, the leaker
* leakers out oily orange goops
* goops contain oil and yuck
* Features coder art

## Why's this needed? 
* goop
* unforeseen consequences

## Changelog

```changelog
(u)Sovexe
(*)Added a new dripper plant mutation, the Leaker. Leakers produce orange goop containing oil and yuck.
```

### CODE REVIEW

File: code/modules/hydroponics/plants_alien.dm [hunk 1]
```diff
@@ -147,6 +147,19 @@ ABSTRACT_TYPE(/datum/plant/artifact)
...
+	override_icon_state = "Leaker"
+	crop = /obj/item/reagent_containers/food/snacks/plant/orangegoop
+	unique_seed = /obj/item/seed/alien/leaker
+	starthealth = 25
+	growtime = 65
+	harvtime = 150
+	cropsize = 3
+	harvests = 4
+	endurance = 10
+	assoc_reagents = list("oil")
```
Comment (Tarmunora) [inline code/modules/hydroponics/plants_alien.dm:161]: not oil *and* yuck?

File: code/modules/hydroponics/plants_alien.dm [hunk 2]
```diff
@@ -147,6 +147,19 @@ ABSTRACT_TYPE(/datum/plant/artifact)
 	endurance = 0
 	assoc_reagents = list("plasma")
 
+/datum/plant/artifact/leaker
+	name = "Leaker"
```
Comment (Tarmunora) [inline code/modules/hydroponics/plants_alien.dm:151]: overall, any reason not for this to be a mutation of the dripper or smth?

File: code/modules/hydroponics/plants_alien.dm (hunk 1 above)
Comment (Sovexe) [reply code/modules/hydroponics/plants_alien.dm:161]:
```suggestion
	assoc_reagents = list("yuck", "oil")
```

File: code/modules/hydroponics/plants_alien.dm (hunk 2 above)
Comment (Sovexe) [reply code/modules/hydroponics/plants_alien.dm:151]: mostly because the puker wasn't a mutation

**Review by Tarmunora — APPROVED** [review]
code looks fine

**Comment by Camryn-Buttes** [comment]
Prepare for unforeseen consequences

**Pull request merged**